cmd/clusterctl/client/yamlprocessor/variableMetaData.go

```diff
@@ -0,0 +1,30 @@
+package yamlprocessor
```

Comment:
This new file needs to have the copyright header:


```
/*
Copyright 2021 The Kubernetes Authors.
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```

Reply:
Also, consider renaming the file to `variable_metadata.go`, it's more go idiomatic